src/Jackett.Common/Indexers/Anidex.cs

```diff
@@ -145,12 +145,14 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             };
 
             // Get specified categories
+            // AniDex throws errors when categories are url encoded. See issue #9727
             var searchCategories = MapTorznabCapsToTrackers(query);
+            var catString = "";
             if (searchCategories.Count > 0)
-                queryParameters.Add("id", string.Join(",", searchCategories));
+                catString = "&id=" + string.Join(",", searchCategories);
```

Comment:
`queryParameters.add("id", string.Join(",", searchCategories))`

Reply:
Huh? @ngosang  basically you are saying that to resolve 9727 we should leave the code as it was?

Reply:
My bad, I'm having a bad day, sorry.